python/nano/src/bigdl/nano/pytorch/gpu_cpu/gpu_cpu.py

```diff
@@ -41,6 +40,14 @@
                       'empty_quantized', 'empty_strided',
                       'frombuffer', 'from_file']
 
+attrs = []
+
+
+def replace_attr(obj, name: str, value):
+    torch_attr = getattr(obj, name)
+    setattr(obj, name, value)
+    attrs.append((obj, name, torch_attr))
```

Comment:
what if `patch_cuda` is called twice? `torch_attr` will be the patched cpu version and the unpatch function will be broken.

Reply:
fixed it